Core/Inc/Systems/External/Can/DBCParser.h

```diff
@@ -0,0 +1,121 @@
+#ifndef RENNSSELAERMOTORSPORT_DBC_H
+#define RENNSSELAERMOTORSPORT_DBC_H
+
+#include <stdint.h>
+
+#define MAX_LINE_LENGTH 256

```

Comment:
Just a few small decleration things:

1. constants can be defined as ```static const int``` to avoid namespace polllution;

2. explicit width types can be added to the ```int``` variables;

3. ```ENDIAN_LITTLE``` and ```ENDIAN_BIG``` can be declared with an enum for better type safety, for example:
```
typedef enum {
    ENDIAN_LITTLE = 0,
    ENDIAN_BIG = 1
} Endianness;
```